guides/gatherers.md

```diff
@@ -203,6 +204,33 @@ facts:
 
 For extra information refer to [trento-project/agent/../gatherers/packageversion_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/packageversion_test.go)
 
+### saphostctrl
+
+This gatherer returns allows access to certain webmethods that `saphostctrl` implements. At this moment only `Ping` and `ListInstances` are supported.
```

Comment:
I wouldn't put the supported web methods in this text. Put some new section something like 
```
Supported web methods: Ping, ListInstances
```
Something like this. We don't need to say at this moment or things like that